client/app/private-database/database/dump/delete/private-database-database-dump-delete.controller.js

```diff
@@ -0,0 +1,28 @@
+angular.module("controllers").controller(
+    "PrivateDatabaseBDDsDumpsDeleteCtrl",
+    class PrivateDatabaseBDDsDumpsDeleteCtrl {
+        constructor ($scope, $stateParams, Alerter, PrivateDatabase) {
+            this.$scope = $scope;
+            this.$stateParams = $stateParams;
+            this.alerter = Alerter;
+            this.privateDatabaseService = PrivateDatabase;
+        }
+
+        $onInit () {
+            this.loading = false;
+            this.bdd = this.$scope.currentActionData.bdd;
+            this.dump = this.$scope.currentActionData.dump;
+            this.$scope.deleteDump = () => this.deleteDump();
+        }
+
+        deleteDump () {
+            this.loading = true;
+            return this.privateDatabaseService
+                .deleteDumpBDD(this.$stateParams.productId, this.bdd.databaseName, this.dump.id)
+                .catch((err) => this.alerter.alertFromSWS(this.$scope.tr("privateDatabase_tabs_dumps_delete_error"), err, this.$scope.alerts.main))
+                .finally(() => {
+                    this.$scope.resetAction();
```

Comment:
`this.loading` is not set at `false`

Reply:
No need to set at false, le wizard is closing at this time, and set at false show the form